### PR TITLE
DVC-7630 [feat]: add enable / disable targeting functionality

### PR DESCRIPTION
**Pull request by kaushalkapasi**
- refactor `dvc targeting get <feature> <env>` to have `<env>` as a param instead of a `--env` flag
- add `dvc targeting enable <feature> <env>`
- add `dvc targeting disable <feature> <env>`

### CODE REVIEW

**Comment by kaushalkapasi** [comment]
Current dependencies on/for this PR:
* main
  * **PR #144** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/cli/144" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/DevCycleHQ/cli/144?utm_source=stack-comment).

File: README.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ USAGE
 * [`dvc projects`](docs/projects.md) - Access Projects with the Management API
 * [`dvc repo`](docs/repo.md) - Manage repository configuration
 * [`dvc status`](docs/status.md) - Check CLI status
-* [`dvc targeting`](docs/targeting.md) - Retrieve Targeting for a Feature from the Management API
+* [`dvc targeting`](docs/targeting.md) - Disable the Targeting for the specified Environment on a Feature
```
Comment (suthar26) [inline README.md:106]: is this accurate?

File: src/commands/targeting/disable.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                [environmentPrompt],
+                {
+                    token: this.authToken,
+                    projectKey: this.projectKey
+                }
+            )
+            responses = { ...responses, environmentKey: userSelectedEnv._environment }
+        }
+
+        const enableTargetingForFeatureAndEnvironment = await disableTargeting(
```
Comment (suthar26) [inline src/commands/targeting/disable.ts:61]: should this be disableTargetingForFeatureAndEnvironment?

File: src/commands/targeting/disable.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            responses = { featureKey: feature }
+        } else {
+            const userSelectedFeature = await inquirer.prompt(
+                [featurePrompt],
+                {
+                    token: this.authToken,
+                    projectKey: this.projectKey
+                }
+            )
+            responses = { featureKey: userSelectedFeature._feature }
```
Comment (suthar26) [inline src/commands/targeting/disable.ts:45]: i think this should be `userSelectedFeature.feature` with the update to the prompt

File: src/commands/targeting/enable.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            responses = { featureKey: feature }
+        } else {
+            const userSelectedFeature = await inquirer.prompt(
+                [featurePrompt],
+                {
+                    token: this.authToken,
+                    projectKey: this.projectKey
+                }
+            )
+            responses = { featureKey: userSelectedFeature._feature }
```
Comment (suthar26) [inline src/commands/targeting/enable.ts:45]: this should be `userSelectedFeature.feature` with the change to the prompt

**Review by suthar26 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kaushalkapasi) [reply README.md]: updated

**Review by suthar26 — APPROVED** [review]

**Pull request merged**